docs/using-the-jdbc-driver/DataSource.md

```diff
@@ -18,15 +18,15 @@ while another might use the method `setUsername` for the same task. See the tabl
-| URL name                    | `setUrlPropertyName`           | The name of the URL property.                                | `String` | No                                                           | `url`                                  |
-| JDBC URL                    | `setJdbcUrl`                   | The URL to connect with.                                     | `String` | No, if there is enough information provided by the other properties that can be used to create a URL. | `jdbc:postgresql://localhost/postgres` |
-| JDBC protocol               | `setJdbcProtocol`              | The JDBC protocol that will be used.                         | `String` | Yes, if the JDBC URL has not been set.                       | `jdbc:postgresql:`                     |
-| Underlying DataSource class | `setTargetDataSourceClassName` | The fully qualified class name of the underlying DataSource class the AWS JDBC Driver should use. | `String` | Yes, if the JDBC URL has not been set.                       | `org.postgresql.ds.PGSimpleDataSource` |
+| Property                    | Configuration Method           | Description                                                                                       | Type     | Required                                                                                              | Example                                |
+|-----------------------------|--------------------------------|---------------------------------------------------------------------------------------------------|----------|-------------------------------------------------------------------------------------------------------|----------------------------------------|
+| Server name                 | `setServerPropertyName`        | The name of the server or host property.                                                          | `String` | Yes, if no URL is provided.                                                                           | `serverName`                           |
+| Database name               | `setDatabasePropertyName`      | The name of the database property.                                                                | `String` | No                                                                                                    | `databaseName`                         |
+| Port name                   | `setPortPropertyName`          | The name of the port property.                                                                    | `String` | No                                                                                                    | `port`                                 |
+| URL name                    | `setUrlPropertyName`           | The name of the URL property.                                                                     | `String` | Only required for MariaDB                                                                             | `url`                                  |
```

Comment:
this is the content change :  Only required for MariaDB    
the rest is formatting change so just ignore

Reply:
I'd keep "No" as a general option and add a note that some drivers, like MariaDb, require some parameters to be included in url so we recommend to provide this parameter.